From: Markus Koschany <apo@debian.org>
Date: Fri, 26 Aug 2016 01:47:41 +0200
Subject: disable tests

Tests cause random build failures which makes them not useful for debugging.

Debian-Bug: https://bugs.debian.org/829377
Forwarded: not-needed
---
 build.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.xml b/build.xml
index f7afdcd..2aed52e 100644
--- a/build.xml
+++ b/build.xml
@@ -27,7 +27,7 @@
         <path refid="javac.classpath"/>
     </path>
 
-    <target name="all" depends="clean,build,test" description="Run clean,build,test"/>
+    <target name="all" depends="clean,build" description="Run clean,build"/>
 
     <target name="build" description="Compile source files">
         <mkdir dir="${dir.classes}"/>
